### PR TITLE
test: Cypress - fixing table test cases 

### PR DESCRIPTION
**Pull request by sharanya-appsmith**
1. Date_column_editing_2_spec.ts - time precision unavailable now in date  column in table - so removed the skipped case
2. fixed mindate and max date skipped case
3. AddNewRow1_spec.js and AddNewRow2_spec.js - combined test cases and added asserts

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV2/Edge_case_spec.js [hunk 1]
```diff
@@ -32,15 +32,6 @@ describe("Table widget v2 edge case scenario testing", function () {
       "have.text",
       "[  1]",
     );
-
-    propPane.TogglePropertyState("Enable multi-row selection", "Off"); //Disable Multi row select
```
Comment (NandanAnantharamu) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV2/Edge_case_spec.js:36]: Is this code not required ?

**Review by NandanAnantharamu — APPROVED** [review]
LGTM

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV2/Edge_case_spec.js (hunk 1 above)
Comment (sharanya-appsmith) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/TableV2/Edge_case_spec.js:36]: Same thing is checked twice. Hence removed the redundant check

**Pull request merged**